shared-components/tl-date-time-card/tl-date-time-card.js

```diff
@@ -0,0 +1,41 @@
+import { LitElement, html, css } from 'lit';
+import { dateTimeStyles } from './tl-date-time-card-css.js';
+
+export class DateTimeCard extends LitElement {
+    static get properties() {
+        return {
+            icon: { type: String },
+            text: { type: String }
+        };
+    }
+
+    static get styles() {
+        return [dateTimeStyles];
+    }
+    constructor() {
+        super();
+        this.icon = '';
+        this.text = '';
+    }
+
+    _formatDates() {
+        const options = { month: 'short', day: 'numeric' };
+
+        var start = new Date(this.tournament.dateStart);
+        var end = new Date(this.tournament.dateEnd);
+        start = start.toLocaleDateString(undefined, options) + this._nth(start.getDate());
+        end = end.toLocaleDateString(undefined, options) + this._nth(end.getDate());
+
+        return start + ' - ' + end;
+    }
+
+    render() {
+        return html`
+            <div class="iconWrapper">
```

Comment:
Why not having the `.iconWrapper` styles applied to `:host`? What is the value added by this wrapper `<div/>`?

Reply:
To be honest, I like to have defined classes, so in case of any change, if we add more elements to component, there is less refactoring to be done.

Reply:
The CSS class name is only valid inside the component (effect limited to the shadow DOM).
With your approach of hiding away in an inner <div/> and having the styles set for that <div/> instead of `host:`, you are removing customization capabilities by the user of the component.

To me adding a <div/> just for styling, especially with the effect of removing external override capabilities is a no-go.

To me, our responsibility is not to make our current or future work easier, it's to first serve the final user, and second to serve the component user (another developer), then the test team, and finally yourself ;)